### PR TITLE
[Peggy] ebrelayer requires passphrase when witnessing a tx at runtime

### PR DESCRIPTION
**Pull request by juniuszhou**
We will use mnemonic for relayer to sign tx.

### CODE REVIEW

File: cmd/ebrelayer/keyring/keyring.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+	"github.com/Sifchain/sifnode/app"
+	"github.com/cosmos/cosmos-sdk/crypto/keys"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/hd"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/tendermint/crypto"
+)
+
+const (
+// AccountAddressPrefix = "sif"
```
Comment (ElliotFriedman) [inline cmd/ebrelayer/keyring/keyring.go:14]: Let's remove commented out code

File: cmd/ebrelayer/keyring/keyring.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+var (
+	// AccountPubKeyPrefix = AccountAddressPrefix + "pub"
```
Comment (ElliotFriedman) [inline cmd/ebrelayer/keyring/keyring.go:18]: Same comment here

File: cmd/ebrelayer/keyring/keyring.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+func (r *KeyRing) Sign(msg []byte) ([]byte, crypto.PubKey, error) {
+	pl, pk, err := r.kb.Sign(r.moniker, r.password, msg)
+	if err != nil {
+		// panic(err)
+		fmt.Println(err)
+		return nil, nil, err
+	}
+	return pl, pk, err
+
+	// currentPubKey, _ := sdk.Bech32ifyPubKey(sdk.Bech32PubKeyTypeAccPub, r.GetAccountFromMnemonic().GetPubKey())
```
Comment (ElliotFriedman) [inline cmd/ebrelayer/keyring/keyring.go:72]: please remove this commented code

**Comment by utx0** [comment]
This is still in progress correct?

**Review by utx0 — APPROVED** [review]
Nice work.

**Pull request merged**